### PR TITLE
JDF-168 Added Contributing guide and removed use cases from readme

### PR DESCRIPTION
**Pull request by rafabene**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,48 @@
-JBoss BOMs 
-====================
+JBoss BOMs Contributing Guide
+=============================
 
-BOMs. There are a number of rules for BOMs:
+BOM's are maven pom.xml files that specify the versions of all runtime dependencies for a given module.  So by importing this BOM, you are specifying the versions of the dependencies required to use the specified stack.
```
Comment (sgilda) [inline CONTRIBUTING.md]: s/BOM's/BOMs (remove apostrophe)
s/maven/Maven (always capitalize Maven)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,6 +1,48 @@
...
+
+Basic Steps
+-----------
+
+To contribute with BOMs, clone your own fork instead of cloning the main BOMs repository, commit your work on topic branches and make pull requests. In detail:
```
Comment (sgilda) [inline CONTRIBUTING.md]: To contribute BOMs, fork the main BOM repository, clone your fork, commit your work on topic branches, and make pull requests.

Here are the steps In detail:

**Comment by sgilda** [comment]
Other than minor comments above, this looks good!

**Comment by sgilda** [comment]
Other than the above, looks good!

File: CONTRIBUTING.md (hunk 2 above)
Comment (rafabene) [inline CONTRIBUTING.md]: /s/In/in ?

**Pull request merged**